iree_tests/configs/config_onnx_cpu_llvm_sync.json

```diff
@@ -612,190 +612,76 @@
     "test_wrap_pad"
   ],
   "expected_run_failures": [
```

Comment:
To get the checks passing on this, https://github.com/nod-ai/SHARK-TestSuite/pull/211 should also be pulled in (can merge that then rebase/sync)

We're also considering trimming the config files in this repo and turning off "strict" XFAILs to make updating the files less of a chore. The general idea is that the config files _in this repo_ are for testing the test suite infrastructure itself, while the configs downstream _in repos like IREE_ are for showing which tests are continuously tested and passing/failing for some given source code and flags.